cmd/module.go

```diff
@@ -36,6 +43,18 @@ func init() {
 	packModuleCmd.Flags().StringVarP(&packCmdPlatform, "platform", "p", "",
 		"the deployment platform; supported plaforms: cf, xsa, neo")
 
+	// sets the flags of of the command zip module
+	zipModuleCmd.Flags().StringVarP(&zipCmdSrc, "source", "s", "",
```

Comment:
if we provide all these flags we have to add them to configuration's command and fill in buildOpts